### PR TITLE
Radiant branding. Removed plugins incompatible with Radiant.

### PR DESCRIPTION
**Pull request by benkno**

### CODE REVIEW

**Comment by benkno** [comment]
Windows and Linux builds are available here https://github.com/benkno/electron-radiant/releases/tag/v0.1.0-alpha.2

File: electroncash_gui/qt/address_list.py [hunk 1]
```diff
@@ -358,37 +358,6 @@ def doCopy(txt):
 
         run_hook('address_list_context_menu_setup', self, menu, addrs)
 
-        # Add Cash Accounts section at the end, if relevant
-        if not multi_select:
```
Comment (RadiantBlockchain) [inline electroncash_gui/qt/address_list.py:362]: I'm curious what this was meant for?

**Review by RadiantBlockchain — APPROVED** [review]
Awesome work with the icons too!

File: electroncash_gui/qt/address_list.py (hunk 1 above)
Comment (benkno) [reply electroncash_gui/qt/address_list.py:362]: This is used by BCH Cash Accounts. If you right click on the address it had these Cash Account menu items.

**Pull request merged**